spectre-meltdown-checker.sh

```diff
@@ -119,31 +33,68 @@ show_usage()
 
 	Options:
 		--no-color			Don't use color codes
-
```

Comment:
Tiny detail: this \n was actually here for a reason: `./spectre-meltdown-checker.sh --invalid` is more readable in that case